src/test/resources/docker-compose-proxy.yaml

```diff
@@ -0,0 +1,27 @@
+---
+version: '3.4'
+services:
+  weaviate:
+    command:
+    - --host
+    - 0.0.0.0
+    - --port
+    - '8080'
+    - --scheme
+    - http
+    image: semitechnologies/weaviate:1.21.0
+    restart: on-failure:0
+    environment:
+      LOG_LEVEL: "debug"
+      QUERY_DEFAULTS_LIMIT: 25
+      AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: 'true'
+      PERSISTENCE_DATA_PATH: "./data"
+      VIRTUAL_HOST: weaviate.local
+      VIRTUAL_PORT: 8080
+  proxy:
+    image: nginxproxy/nginx-proxy:alpine
+    ports:
+      - "8090:80"
```

Comment:
This part can be removed, bc in all other docker compose files we don't expose ports to host